src/OrbitGl/CaptureViewElement.h

```diff
@@ -92,12 +80,16 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
   bool picked_ = false;
   bool visible_ = true;
 
+  void Draw(Batcher& batcher, TextRenderer& text_renderer, const DrawContext& draw_context);
+  void UpdatePrimitives(Batcher* batcher, TextRenderer& text_renderer, uint64_t min_tick,
```

Comment:
For consistency, we should choose Batcher and TextRenderer either both as pointers or both as references.

Reply:
I agree. But I would do this in a separate PR and change `Batcher*` to `Batcher&` - if I do this in the same PR it will touch a lot of existing code once again.

Reply:
Nice to hear :)